_pages/en/capabilities.md

```diff
@@ -693,46 +705,46 @@ Aim for a Westrum culture metric > 35
   b. Agree  
   c. Undecided  
   d. Disagree  
-  e. Strongly Disagree  
+  e. Strongly Disagree (Ideal)  
 
 - My  team has the resources to engage in informal learning and the space to explore ideas  
-  a. Strongly Agree  
+  a. Strongly Agree (Ideal)  
   b. Agree  
   c. Undecided  
   d. Disagree  
   e. Strongly Disagree  
 
 - My team has a regular cadence of opportunities for employees to share their knowledge  
-  a. Strongly Agree  
+  a. Strongly Agree (Ideal)  
   b. Agree  
   c. Undecided  
   d. Disagree  
   e. Strongly Disagree  
 
 - In my team resources are available for continued education  
-  a. Strongly Agree  
+  a. Strongly Agree (Ideal)  
   b. Agree  
   c. Undecided  
   d. Disagree  
   e. Strongly Disagree  
 
 - My team has regular lunchtime meetings where one person presents about a project they are working on in a new tech, or something they are investigating  
```

Comment:
What's the source for this one? Seems oddly specific rather than outcome driven

Ex: If they dont do this, but they still have ways to discuss current projects and share information that also results in the output we're looking for from this question, then did it really need to be a lunch time meeting?